src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const Modal = ({ isShowing, hide }: { isShowing: boolean; hide: any }) =>
+  isShowing
+    ? ReactDOM.createPortal(
+        <React.Fragment>
```

Comment:
```suggestion
        <>
```
There is a syntax sugar for fragment. same thing, just looks better)